### PR TITLE
Foxtrot cryomarines spawn as freed mobs if not taken

### PR DESCRIPTION
**Pull request by ihatethisengine**
# About the pull request

Foxtrot cryomarines spawn as freed mobs if not taken.

# Explain why it's good for the game

It sucks when only 3 cryomarines spawn in because there is not enough candidates. Now people can take cryomarines later.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl: ihatethisengine
add: foxtrot cryomarines spawn as freed mobs if not taken.
/:cl:

### CODE REVIEW

**Comment by cm13live** [comment]
This pull request has been mentioned on **CM-SS13**. Please provide feedback here:

https://forum.cm-ss13.com/t/foxtrot-minimum-deployment-number/3623/2

File: code/datums/emergency_calls/cryo_marines.dm [hunk 1]
```diff
@@ -12,65 +12,80 @@
...
+	spawn_max_amount = TRUE
 
 /datum/emergency_call/cryo_squad/spawn_candidates(announce, override_spawn_loc, announce_dispatch_message)
 	var/datum/squad/marine/cryo/cryo_squad = RoleAuthority.squads_by_type[/datum/squad/marine/cryo]
 	leaders = cryo_squad.num_leaders
 	. = ..()
-	if(length(members))
-		shipwide_ai_announcement("Successfully deployed [length(members)] Foxtrot marines.")
+	// if(length(members))
+	shipwide_ai_announcement("Successfully deployed [mob_max] Foxtrot marines, of which [length(members)] ready for duty.")
```
Comment (Zonespace27) [inline code/datums/emergency_calls/cryo_marines.dm:22]:
```suggestion
	shipwide_ai_announcement("Successfully deployed [mob_max] Foxtrot marines, of which [length(members)] are ready for duty.")
```

File: code/datums/emergency_calls/cryo_marines.dm [hunk 2]
```diff
@@ -12,65 +12,80 @@
...
 	var/leaders = 0
+	spawn_max_amount = TRUE
 
 /datum/emergency_call/cryo_squad/spawn_candidates(announce, override_spawn_loc, announce_dispatch_message)
 	var/datum/squad/marine/cryo/cryo_squad = RoleAuthority.squads_by_type[/datum/squad/marine/cryo]
 	leaders = cryo_squad.num_leaders
 	. = ..()
-	if(length(members))
-		shipwide_ai_announcement("Successfully deployed [length(members)] Foxtrot marines.")
+	// if(length(members))
```
Comment (Zonespace27) [inline code/datums/emergency_calls/cryo_marines.dm:21]: ?

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
Lgtm otherwise

File: code/datums/emergency_calls/cryo_marines.dm (hunk 2 above)
Comment (ihatethisengine) [inline code/datums/emergency_calls/cryo_marines.dm:21]:
```suggestion
```

Comment (ihatethisengine) [reply code/datums/emergency_calls/cryo_marines.dm]: forgot to delete

**Review by Zonespace27 — APPROVED** [review]
Good for tm

**Review by harryob — APPROVED** [review]

**Pull request merged**